src/main/java/com/cloudbees/jenkins/plugins/kubernetes_credentials_provider/KubernetesCredentialProvider.java

```diff
@@ -163,6 +171,16 @@ public void stopWatchingForSecrets() {
         if (ACL.SYSTEM.equals(authentication)) {
             ArrayList<C> list = new ArrayList<>();
             for (IdCredentials credential : credentials.values()) {
+                if (itemGroup != Jenkins.getInstanceOrNull()) {
+                    String scope = '/' + itemGroup.getUrl();
```

Comment:
why adding /. any scope added at the secret should be from the root shouldn't it?
or are you expecting a scope in the k8s secret to be GLOBAL or SYSTEM as well as an itemgroup?